src/dbt/kipptaf/models/people/staging/stg_people__manager_pm_score_averages.sql

```diff
@@ -0,0 +1,167 @@
+with
+    measures_long as (
+        select
+            observer_employee_number,
+            academic_year,
+            form_term,
+            row_score_value,
+            -- trunk-ignore-begin(sqlfluff)
+            case
```

Comment:
are these codes available from schoolmint?

Reply:
no, but we set up a regex for it